breba_app/my_cl_app.py

```diff
@@ -201,17 +200,16 @@ async def window_message(message: str | dict):
 
     if method == "to_builder":
         await to_builder(user_name, product_id, message.get("body", "INVALID REQEUST, something went wrong"),
-                         builder_completed,
-                         ask_user, process_generator_message)
+                         builder_completed, process_generator_message, ask_user)
     elif method == "to_generator":
         await to_generator(user_name, product_id, message.get("body", "INVALID REQEUST, something went wrong"),
-                           builder_completed, process_generator_message, ask_user)
+                           generator_completed, process_generator_message, ask_user)
```

Comment:
This is not correct, should be reverted back to "builder_completed". Sorry a bit inconsistent naming, but generator_completed is just a helper function that gets called by process_generator_message.